client/components/imgSelector.jsx

```diff
@@ -84,11 +75,14 @@ class ImageSelector extends React.Component {
           <Col size={.75}>
             {images.map((pic) => {
               if (pic.url === active) {
-                return <Thumbnail src={pic.url} onClick={() => this.changePic(pic.url)} key={pic.id} style={{ boxShadow: 'inset 0px -3px red' }} />;
+                return <Thumbnail src={pic.url} onClick={() => this.changePic(pic.url)} key={pic.id} style={{ boxShadow: 'inset 0px -3px rgb(218, 41, 28)' }} />;
               }
```

Comment:
Is there a way to these inline styles in the styled component above? Maybe with conditionals?

Reply:
That's a good point, I'll look into that.